src/pages/site/[uuid]/tabs/Overview.tsx

```diff
@@ -137,200 +173,201 @@ const SiteOverviewTab = ({ site }: SiteOverviewTabProps) => {
-                    }}
-                  >
-                    <Icon name={IconNames.DOWNLOAD_PA} className="h-4 w-4" />
-                    &nbsp; {t("Download")}
-                  </Button>
-                  {isMonitoring && (
-                    <Menu placement={MENU_PLACEMENT_BOTTOM_BOTTOM} menu={itemsSubmitPolygon}>
-                      <Button variant="primary" className="" onChange={() => {}}>
-                        {t("SUBMIT Polygons")}
-                      </Button>
-                    </Menu>
-                  )}
-                </div>
+            <div className="flex w-full">
+              <div className="grid w-[50%] grid-cols-2 content-start gap-x-8 gap-y-7 pr-20">
+                <When condition={isPPC}>
+                  <GoalProgressCard label={t("Workday Count (PPC)")} value={site.workday_count} />
+                </When>
+                <GoalProgressCard label={t("Hectares Restored Goal")} value={site.hectares_to_restore_goal} />
+              </div>
+              <div>
+                <GoalProgressCard
+                  label={t("Trees restored")}
+                  hasProgress={false}
+                  items={[
+                    { iconName: IconNames.TREE_CIRCLE, label: t("Trees Planted"), value: site.trees_planted_count },
+                    { iconName: IconNames.LEAF_CIRCLE, label: t("Seeds Planted"), value: site.seeds_planted_count },
+                    {
+                      iconName: IconNames.REFRESH_CIRCLE,
+                      label: t("Trees Regenerating"),
+                      value: site.regenerated_trees_count
+                    }
+                  ]}
+                  className="flex-1"
+                />
               </div>
             </div>
-            <SiteArea sites={site} setEditPolygon={setEditPolygon} editPolygon={editPolygon} />
           </PageCard>
-        </PageColumn>
-      </PageRow>
-      <PageRow>
-        <PageColumn>
-          <PageCard
-            title={t("Project Monitoring")}
-            tooltip="Lorem ipsum dolor sit amet, urna neque viverra justo nec ultrices dui sapien eget mi proin sed libero."
-          >
-            <div className="flex items-center justify-between text-darkCustom">
-              <Text variant="text-14-light" className="w-[65%]">
-                {t(`Select all or specific sites to view remote sensing analytics such as tree counts, NDVI, and other
-                metrics useful for assessing the impact of the restoration effort.`)}
-              </Text>
-              <div className="relative w-[25%]">
-                <Dropdown
-                  containerClassName="w-full"
-                  placeholder={t("All Polygons")}
-                  options={[
+        </PageRow>
+        <PageRow>
+          <PageColumn>
+            <PageCard
+              title={t("Site Area")}
+              tooltip="Lorem ipsum dolor sit amet, urna neque viverra justo nec ultrices dui sapien eget mi proin sed libero."
```

Comment:
Please make sure we're not including any mocked / test data in merges to staging. Also, if we have real text to bring in for this tooltip, it'll need to be wrapped in `t()`

Reply:
tooltip removed